scripts/get_community_assigned_issues.py

```diff
@@ -0,0 +1,73 @@
+# Prints a list of links to all issues that are assigned
+# to external contributors.
+# This is not used by any workflows, and meant to be used locally
+# whenever need arises. The script requires `GITHUB_TOKEN`
+# environment variable with 'repo' and 'read:org' permissions. 
+
+import os
+import requests
+
+ORG = "learningequality"
+REPOS = ["kolibri", "studio", "kolibri-design-system", "le-utils", ".github","ricecooker"]
+
+TOKEN = os.getenv("GITHUB_TOKEN")
+if not TOKEN:
+    raise EnvironmentError("Please set the GITHUB_TOKEN environment variable with 'repo' and 'read:org' permissions.")
+
+BASE_URL = "https://api.github.com"
+HEADERS = {
+    "Authorization": f"token {TOKEN}",
+    "Accept": "application/vnd.github+json"
+}
+
+def get_team_members(org):
+    """Fetch all team members for the organization."""
+    team_members = set()
+    url = f"{BASE_URL}/orgs/{org}/members"
+    while url:
+        response = requests.get(url, headers=HEADERS)
+        response.raise_for_status()
+        members = response.json()
+        team_members.update(member["login"] for member in members)
+        url = response.links.get("next", {}).get("url")
+    return team_members
+
+def get_issues_for_repo(org, repo):
+    """Fetch all issues for a specific repository."""
+    issues = []
+    url = f"{BASE_URL}/repos/{org}/{repo}/issues?state=open"
+    while url:
```

Comment:
Or we could also extract the fetch functionality something like this?
```
def fetch_paginated_results(url):
    results = []
    while url:
        response = requests.get(url, headers=HEADERS)
        response.raise_for_status()
        results.extend(response.json())
        url = response.links.get("next", {}).get("url") 
    return results
 ```
 This way we could use this in any other future functions as well.